.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -47,7 +47,7 @@ body:
         - ParadeDB Docker Image
         - ParadeDB Helm Chart
         - ParadeDB pg_search Extension
-        - ParadeDB pg_lakehouse Extension
+        - ParadeDB pg_analytics Extension
```

Comment:
This should be removed, all `pg_analytics` issues should be opened in the `pg_analytics` repo

Reply:
We'll do that in a separate PR once the conversion is complete

Reply:
Jk, I fixed it right away.